src/postw90/gyrotropic.F90

```diff
@@ -484,7 +484,7 @@ subroutine gyrotropic_get_k_list(kpt, kweight, &
       pw90common_fourier_R_to_k_vec
     use w90_wan_ham, only: wham_get_eig_deleig, wham_get_D_h
 
-    use w90_get_oper, only: HH_R, SS_R, AA_R
+    use w90_get_oper, only: HH_R, AA_R
```

Comment:
Actually, HH_R is also not needed here

Reply:
I removed it: 93ac701. ( I also moved `pw90common_fourier_R_to_k_vec` outside the loop: my mistake with changing from `fourier_new` to `fourier_vec`.)

There is a separate branch where I removed unused variables and USE statements using the `-check all -warn all` flag of ifort compiler, but I thought it would be better to have a separate PR for that.

See https://github.com/jaemolihm/wannier90/compare/use_ws_opt...jaemolihm:wigner_opt.